Dockerfile

```diff
@@ -64,18 +68,18 @@ CMD ["php-fpm"]
 ENV COMPOSER_ALLOW_SUPERUSER=1
 ENV PATH="${PATH}:/root/.composer/vendor/bin"
 
-COPY --from=composer/composer:2-bin --link /composer /usr/bin/composer
+COPY --from=composer --link /composer /usr/bin/composer
 
 # prevent the reinstallation of vendors at every changes in the source code
-COPY composer.* symfony.* ./
+COPY --link composer.* symfony.* ./
```

Comment:
Is this safe? Will it not prevent creating a specific cache layer just with the vendors?

Reply:
I think, according to what is indicated in the docker documentation, it seems to be : https://docs.docker.com/engine/reference/builder/#benefits-of-using---link